package.json

```diff
@@ -70,11 +70,12 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^4.0.1",
     "node-sass": "^4.13.0",
-    "react": "^16.11.0",
-    "react-dom": "^16.11.0",
+    "react": ">=16.11.0",
+    "react-dom": ">=16.11.0",
     "react-tools": "^0.13.2",
     "react-transform-catch-errors": "^1.0.0",
     "react-transform-hmr": "^1.0.1",
+    "create-react-class":"^15.7.0",
```

Comment:
where is this used? And please squash your commits.

Reply:
Sorry i left my old account should i push it from here again  or ?

Reply:
sure. maybe create a new PR. the commits looks naive